ManagedApp/mainTemplate.json

```diff
@@ -156,6 +156,19 @@
       ],
       "location": "[resourceGroup().location]",
       "resources": [
+		{ 
+			"apiVersion": "2018-02-01",
+			"name": "pid-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX", // use your generated GUID here for resource tracking as described at https://aka.ms/aboutinfluencedrevenuetracking and remove these comments
```

Comment:
Change format to be consistent with the convention in the rest of the file - 

"name": "<valid resource tracking guid>"

Reply:
the format is taken from their docs so if they read that URL, did not want customers to see anything different vs related documentation. Also the format is: pid-<guid> not a <guid>

Reply:
Merged both version, I think it is consistent now and also gives necessary link